cinema/models.py

```diff
@@ -1,10 +1,36 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
```

Comment:
The `max_length` for `first_name` and `last_name` in the `Actor` model should be 100 to match the migration file, which specifies a `max_length` of 100 for these fields.